### DISCUSSION TITLE
is it possible to do click to call triggering?

### DISCUSSION THREAD

**Discussion by bbakirtas** [InnovateAsterisk/Browser-Phone#391] [Q&A]
This software is very successful. What I want to do is integrate it into my own software and click and call. Can this make a call when he clicks on a number or the call button in the example table?
 like in softphones; sip:12345

**Comment by InnovateAsterisk** [comment]
The closest thing I have to that is: https://www.innovateasterisk.com/phone/popup.html
Once you click link the phone appears as an iframe window.

**Reply by bbakirtas** [reply]
yes, it is very successful. but my goal is to make calls without using it. I think something like api is needed for this. like index.php?call=12345

**Reply by solucionesuno** [reply]
+1 @InnovateAsterisk is any method to run a call directly when open the webphone?
